shell/platform/windows/public/flutter_windows.h

```diff
@@ -203,6 +203,28 @@ FLUTTER_EXPORT void FlutterDesktopEngineSetNextFrameCallback(
     VoidCallback callback,
     void* user_data);
 
+typedef struct {
+  size_t cb_size;
+  HWND parent_window;
+  const char* platform_view_type;
+  void* user_data;
+  int64_t platform_view_id;
```

Comment:
I know we don't do it on other platforms (yet), but I think we should consider extracting a typedef for PlatformView IDs for the same reason as mentioned for our internal APIs.